src/error-page.tsx

```diff
@@ -1,12 +1,12 @@
 import { useRouteError } from "react-router-dom";
 
-interface RouteError {
+interface RouteErrorProps {
   statusText: string;
   message: string;
 }
 
 export function ErrorPage() {
-  const error = useRouteError() as RouteError;
+  const error = useRouteError() as RouteErrorProps;
```

Comment:
Do we need `as RouteError`? Doesn't the hook have assigned type from the lib itself?

Reply:
No it doesn't. Already confirmed this with Lamsh. I also thought it was strange since the type for useRouteError is unknown.